tests/test_two_sided_market.py

```diff
@@ -8,15 +8,26 @@
 from math import isclose
 from unittest.mock import MagicMock, patch
 from d3a_interface.constants_limits import ConstSettings
+from d3a_interface.enums import BidOfferMatchAlgoEnum
 from d3a_interface.matching_algorithms import (
     PayAsBidMatchingAlgorithm, PayAsClearMatchingAlgorithm
 )
 from d3a.d3a_core.exceptions import (
     BidNotFoundException, InvalidBid, InvalidBidOfferPairException, InvalidTrade, MarketException)
 from d3a.events import MarketEvent
+from d3a.models.area import Area
 from d3a.models.market import Bid, Offer
 from d3a.models.market.market_structures import TradeBidOfferInfo, Trade
 from d3a.models.market.two_sided import TwoSidedMarket
+from d3a.d3a_core.singletons import bid_offer_matcher
+
+
+@pytest.fixture
+def pac_area():
```

Comment:
@hannesdiedrich You left a comment lately about not able to find its usage, somehow, I can't find that comment now. Anyway heres my answer
It was added as part of [unittest](https://github.com/gridsingularity/d3a/pull/1195/commits/c625d77c200cddfb35c9a70e3174121bbf408684#diff-e443008143292bdfa0c5f6d4d871a132e7d974f5078bc4493649dbfa5b2506e4R510) when market clearing state was associated to its area. No more valid as its now has been part of bid offer matcher's singleton.